spec/services/scrapers/high_priority_spec.rb

```diff
@@ -4,17 +4,34 @@
 
 RSpec.describe Scrapers::HighPriority do
   describe '#perform' do
-    it 'add specs' do
-      county = create(:county, name: 'Tulsa')
-      test_case = create(:court_case, closed_on: 7.days.ago, county: county)
+    # let(:test_case) {create(:court_case, closed_on: 7.days.ago, county: county)}
+    before(:each) do
+      @county =  create(:county, name: 'Tulsa')
+      @test_case = create(:court_case, closed_on: 7.days.ago, county: @county)
+      @data = nil
```

Comment:
I don't think you'd actually be able to reference these variables anywhere, that's why we use `let!` instead
But looks much better.